### ISSUE TITLE
Check Roles/Access Across the r-hyperspec Family

### ISSUE THREAD

**Issue by bryanhanson** [r-hyperspec/r-hyperspec.github.io#6]
We need to make sure that the `r-hyperspec` team has "write" access to all repos, that Claudia has "admin" access to all repos, and then each person is given additional access as appropriate.  Individuals don't need to be listed unless they have access above that level assigned to the team. So if I am listed as "write" as is the team, then my individual listing can be deleted.

I am going to assign this to Claudia as I it is ultimately her decision.

**Comment by GegznaV** [comment]
I agree with the ideas Bryan wrote.

**Comment by cbeleites** [comment]
I went through the repos and 

- made sure r-hyperspec has "write" rights everywhere (in one case I demoted from "admin")
- deleted a bunch of redundant (individual write) entries
- I was displayed with "write" rights for some, but a closer inspection showed this was actaully "admin" 
- It looks as if I automatically have "admin" rights everywhere since I'm owner of the organization.
- In different repos different other people also have "admin" rights, but I think this is OK for now. 

I think we'll need to have another go possibly at the end of the GSoC. I therefore leave this issue open, but put a new label "end-of-GSoC"
